client/my-sites/stats/mobile-promo-cards/index.jsx

```diff
@@ -0,0 +1,82 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { MobilePromoCard } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import wordpressSeoIllustration from 'calypso/assets/images/illustrations/wordpress-seo-premium.svg';
+import PromoCardBlock from 'calypso/blocks/promo-card-block';
+import DotPager from 'calypso/components/dot-pager';
+
+import './style.scss';
+
+const EVENT_YOAST_PROMO_VIEW = 'calypso_stats_wordpress_seo_premium_banner_view';
+const EVENT_MOBILE_PROMO_VIEW = 'calypso_stats_traffic_mobile_cta_jetpack_view';
+
+export default function MobilePromoCardWrapper( { isJetpack, isOdysseyStats, slug } ) {
+	// Yoast promo is disabled for Odyssey & self-hosted.
+	// Mobile apps promos are always shown.
+	const showYoastPromo = ! isOdysseyStats && ! isJetpack;
+	const showBothPromoCards = showYoastPromo;
```

Comment:
```suggestion

```

I think the variable `showYoastPromo` with annotations is enough to explain the intention.